x/orderbook/model.go

```diff
@@ -31,19 +32,18 @@ func (m *Market) Copy() orm.CloneableData {
 
 // Validate is always succesful
 func (m *Market) Validate() error {
-	// if err := m.Metadata.Validate(); err != nil {
-	// 	return err
-	// }
-	if err := isGenID(m.ID, true); err != nil {
-		return err
-	}
-	if err := m.Owner.Validate(); err != nil {
-		return errors.Wrap(err, "owner")
-	}
+	var errs error
+
+	//errs = errors.AppendField(errs, "Metadata", m.Metadata.Validate())
```

Comment:
Why metadata validation is not included? If you use it, you should validate it. Otherwise, maybe it is better to remove the `Metadata` attribute.

Reply:
Tutorial uses `tutorial/morm` package which does not provide migration mechanism. Uncommenting this totally depends on https://github.com/iov-one/tutorial/issues/16. I can take a look at it in a different issue